site/content/blog/getting-started-with-ytt.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Getting Started With ytt, Part 1"
+slug: getting-started-with-ytt
+date: 2022-10-12
+author: Varsha Munishwar
+excerpt: "Are you new to ytt and wondering where to start?
+Here is an easy step-by-step tutorial that can introduce you to ytt and help you get started quickly.
+"
+image: /img/ytt.svg
+tags: ['ytt', 'getting started', 'tutorials']
+---
+
+#### Welcome to the series of ytt getting started tutorials!
+
+The part 1 of this series introduces you to [ytt](https://carvel.dev/ytt/) and helps you get started quickly.
+It is an easy step-by-step tutorial that you can follow along and see ytt in action on the playground as well as on the CLI.
+
+We will cover the following topics:
+- Introduction to ytt
+- What problems is ytt solving?
+- See ytt in action on interactive playground and CLI
+
+####  Video link - [Getting started tutorial - Part 1](https://youtu.be/DvApsPy0IrI)
+{{< youtube id="DvApsPy0IrI" title="Getting started tutorial - Part 1" >}}
+
+The key moments/timestamps are available if you watch on youtube. Please click on "more" and "view all".
+
+Here is the YAML code to follow along on the [ytt playground](https://carvel.dev/ytt/#playground
+)
+
+#### 1) Simple Hello World!
+```yaml
+#! "Welcome to ytt tutorial series"
+tutorials:
+          title: "Hello world, welcome to ytt!"
+           #! "Welcome to ytt tutorial series" 
```

Comment:
I know this is valid YAML but it looks strange _to me_. Is there any value that we get by having a larger indentation?

Reply:
This was indented purposefully to show ytt can handle these. I will add context to it in the blog post